### PR TITLE
Migrate Workspace, ExternalModules, Renderer packages

### PR DESCRIPTION
**Pull request by dmitriyslaym**
https://github.com/capsulajs/capsulajs/issues/25

Release notes:
1) Migrated workspace, cdn-emulator, service-renderer, service-configuration, service-selector.
2) Added documentations in each package and updated main read.me and templates

### CODE REVIEW

File: packages/cdn/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@capsulajs/capsulahub-cdn",
```
Comment (stephanebenayoun) [inline packages/cdn/package.json]: We need a different name for this package.
maybe `capsulahub-cdn-mock` or `capsulahub-cdn-emulator` or something like this, but capsulahub-cdn is not good (and as we agreed to publish all packages we need to take care to change it before first publish)

File: packages/service-renderer/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Service Renderer
+
+Service extension, that renders all the web-components, that are registered in Workspace.
+
+# WorkspaceConfiguration example
+
+```json
+  "services": {
+      "serviceName": "RendererService",
+      "path": "http://cdnhost.com/services/RendererService.js",
```
Comment (stephanebenayoun) [inline packages/service-renderer/README.md]: Why not putting the real url of the service here ? (master or develop link)

File: packages/service-configuration/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- [fetch()](#fetch--api)
+
+- [save()](#save--api)
+
+- [More](https://github.com/capsulajs/configuration-service/blob/develop/src/api/ConfigurationServiceTypes.ts)
+
+## Usage
+
+```bash
+yarn add @capsulajs/capsulahub-core-workspace @capsulajs/capsulahub-extensions-configuration
```
Comment (stephanebenayoun) [inline packages/service-configuration/README.md]: Need to be updated

File: packages/service-configuration/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+```javascript
+# ./config.js
+
+module.exports = {
+  name: 'my-app',
+  services: [
+    {
+      serviceName: 'configurationService',
+      path: 'https://configuration-service-http.genesis.om2.com',
```
Comment (stephanebenayoun) [inline packages/service-configuration/README.md]: What is this url ?

Comment (dmitriyslaym) [reply packages/service-configuration/README.md]: I copy/pasted this one, don't know

**Pull request merged**